Public/Src/Utilities/Native/IO/Windows/FileSystem.Win.cs

```diff
@@ -2911,6 +2911,17 @@ private bool TryGetFileAttributesViaFindFirstFile(string path, out FileAttribute
             }
         }
 
+        public static bool IsPathReparseDirectory(string path)

```

Comment:
>public static bool IsPathReparseDirectory(string path) [](start = 8, length = 54)

No need to change anything here. We have in `FileUtilities` a function called `IsDirectorySymlinkOrJunction`.